model/src/main/kotlin/OrtResult.kt

```diff
@@ -157,6 +157,33 @@ data class OrtResult(
         return analyzerErrors.zipWithDefault(scannerErrors, emptySet()) { left, right -> left + right }
     }
 
+    /**
+     * Return the set of all project or package identifiers in the result, optionally [including those of sub-projects]
+     * [includeSubProjects].
+     */
```

Comment:
What is the use case for excluding project which other projects depend on?

Reply:
The use-case is that for sub-projects we may not want to perform certain checks, like verifying that they have a detected license.